x-pack/plugins/reporting/common/constants.ts

```diff
@@ -7,12 +7,15 @@
 export const JOB_COMPLETION_NOTIFICATIONS_SESSION_KEY =
   'xpack.reporting.jobCompletionNotifications';
 
-export const API_BASE_URL = '/api/reporting';
+export const API_BASE_URL = '/api/reporting'; // "Generation URL" from share menu
+export const API_BASE_URL_V1 = '/api/reporting/v1'; //
+
+export const CONTENT_TYPE_CSV = 'text/csv';
 
 export const WHITELISTED_JOB_CONTENT_TYPES = [
   'application/json',
   'application/pdf',
-  'text/csv',
+  CONTENT_TYPE_CSV,
```

Comment:
Any reason it's uppercased here? Not sure if there's a semantic reason... my guess is not

Reply:
it's also exported as a const and used elsewhere